.azure-devops/templates/run-all-tests.yml

```diff
@@ -0,0 +1,8 @@
+steps:
+- task: AzureCLI@2
+  displayName: 'Run all tests'
+  inputs:
+    azureSubscription: AzIoTCLIService
+    scriptType: bash
+    scriptLocation: inlineScript
+    inlineScript: pytest --junitxml "TEST-results.xml"
```

Comment:
Can we please change the script syntax slightly like (example not including coverage)

`pytest -vv azext_iot/tests/ --junitxml=junit/test-iotext-full-results.xml`